cmd/user-cluster-controller-manager/main.go

```diff
@@ -101,6 +101,7 @@ type controllerRunOptions struct {
 	isKonnectivityEnabled        bool
 	konnectivityServerHost       string
 	konnectivityServerPort       int
+	konnectivityKeepaliveTime    string
```

Comment:
Nit: why is this not a duration type, e.g. `time.Time` or `metav1.Time`?

Reply:
Because Konnectivity's `keepalive-time` option is a `flag.DurationVar`  .
`ParseDuration`  expect string as input, that's why it is a string as we just proxy the parameter to konnectivity.

https://pkg.go.dev/time#ParseDuration